cli/src/main/java/io/aklivity/zillabase/cli/internal/commands/asyncapi/list/ZillabaseAsyncapiListCommand.java

```diff
@@ -29,50 +31,45 @@
     description = "List AsyncAPI specifications")
 public final class ZillabaseAsyncapiListCommand extends ZillabaseAsyncapiCommand
 {
-    @Option(name = {"--id"},
-        description = "AsyncAPI specification identifier")
-    public String id;
-
-    @Option(name = {"-u", "--url"},
-        description = "Admin Server URL")
-    public URI serverURL;
-
     @Option(name = {"-v", "--verbose"},
         description = "Show verbose output")
     public boolean verbose;
 
     @Override
     protected void invoke()
     {
-        String response;
         HttpClient client = HttpClient.newHttpClient();
-
-        if (id != null)
-        {
-            response = sendHttpRequest(String.format(ASYNCAPI_ID_PATH, id), client);
-        }
-        else
-        {
-            response = sendHttpRequest(ASYNCAPI_PATH, client);
-        }
+        String response = sendHttpRequest(ASYNCAPI_PATH, client);
 
         if (response != null)
         {
-            System.out.println(response);
+            try
+            {
+                ObjectMapper mapper = new ObjectMapper();
+                JsonNode jsonNode = null;
+                jsonNode = mapper.readTree(response);
+                JsonNode artifactsArray = jsonNode.get("artifacts");
+                System.out.println("Registered AsyncAPI Spec:");
+
+                for (JsonNode artifact : artifactsArray)
+                {
+                    String id = artifact.get("id").asText();
+                    System.out.println(id);
+                }
```

Comment:
This extraction of the `id` from the JSON response body will also need to be abstracted out, so please include this in the same bug filed for above `X-Registry-ArtifactId` in `add` command.